### DISCUSSION TITLE
🏈 Remastered (Release candidate 1)

### DISCUSSION THREAD

**Discussion by swiftyfinch** [swiftyfinch/Rugby#219] [Releases]
You can find the previous beta release [here](https://github.com/swiftyfinch/Rugby/releases/tag/2.0.0b5).
Use this command to update Rugby from `2.0.0b2+`:
```
rugby update --beta
```
Please, check out the 📖 [readme](https://github.com/swiftyfinch/Rugby/tree/2.0.0b6#readme). You can find there all information.

### Changelog

🐛 Bug fixes;
🚀 Sped up deleting targets and patching products (upto 4x times in some cases);
✂️ Add `strip` option which remove debugging symbols ([Cache.md#disable-debug-symbols-generation](https://github.com/swiftyfinch/Rugby/blob/main/Docs/Cache.md#disable-debug-symbols-generation)).

<br>
<img src="https://user-images.githubusercontent.com/64660122/209442704-3f636690-0340-46b5-a3b9-1d9e3779ba82.gif" width=400>

<hr /><em>This discussion was created from the release <a href='https://github.com/swiftyfinch/Rugby/releases/tag/2.0.0b6'>🏈 Remastered (Release candidate 1)</a>.</em>

**Comment by mlch911** [comment]
Nice work.
BTW, I don't find any code about rugby 2.

**Reply by swiftyfinch** [reply]
@mlch911 Yeah, there is [the roadmap](https://github.com/swiftyfinch/Rugby/tree/2.0.0b6#release-roadmap-). And I'm going to open source after the production release.
